packages/fxa-react/configs/tailwind.js

```diff
@@ -10,25 +10,12 @@ module.exports = {
   content: ['./src/**/*.tsx', './public/index.html'],
   theme: {
     extend: {
-      zIndex: {
-        1000: '1000',
-        9999: '9999',
-      },
```

Comment:
These are not included by default by tailwind, but do not seem to be currently used.

Reply:
Did a quick <kbd>grep</kbd> to try and see the state of the Z's&hellip;

```sh
git grep -EIoh "\bz-([[:digit:]]|auto)+" packages | sort | uniq

# z-0
# z-1
# z-10
# z-20
# z-50
```

The `z-1` is interesting, I didn't see it in https://tailwindcss.com/docs/z-index.

```sh
git grep -EI "\bz-1\b" packages | cat

# packages/fxa-payments-server/src/components/SubscriptionTitle/index.tsx:      className={`subscription-title bg-white shadow-sm shadow-grey-300 text-center mt-0 mx-4 mb-auto pt-5 px-4 pb-px border-y-auto z-1 row-start-1 row-end-2 tablet:mx-0 ${className}`}
```

And here's all the custom zIndex'es:
```sh
git grep -EIoh "\bz-\[([[:digit:]])+\]" packages | sort | uniq

# z-[1000]
# z-[9999]
```

Reply:
Thanks for the `grep` check! That `z-1` is curious indeed, it's not being rendered at all since it's not provided by tailwind. Perhaps @xlisachan would know if this should be a different z-index or can safely be removed?

Since the z-indexes of 1000 and 9999 are marked as custom they can safely be excluded from the config file, I think.

Reply:
`z-1000` and `z-9999` were first introduced in #13759 and #13920 but in both cases, @xlisachan and I ended up including them as arbitrary values, so unless Lisa has any objections, I think they can be safely removed from the config. I checked and other than #13962, which also uses `1000` as an arbitrary value, I found no other occurrences.

While the numbers seem absurdly large (especially `9999` and we are all aligned on that), they were done consciously, for a few reasons: [1]

1. Both `1000` and `9999` are the original `z-index` values.
2. As per the epic, we were explicitly directed to focus on direct conversion and limit non-conversion in case it interfered with progress and caused tickets to go out of scope. We make style adjustments that improve responsiveness and a11y, and follow mobile-first (as you have seen, it was desktop-first before). Where possible, we try to make further improvements to structure, reduce DRY, and improve overall UX, but only if we have bandwidth, the change(s) would benefit multiple components/routes (and therefore make our lives easier in turn), and can be easily slipped in without falling out of scope.
3. Both components are some of the earliest to get converted, so a lot of other components still had individual `index.scss` files with various (and most likely randomly chosen) `z-index` values that aren't among, and greater than, the TW defaults. Both `LoadingOverlay` in particular _needs_ to be indexed above other components. It was out of scope in all three PRs to figure out the lowest possible `z-index` value(s) needed to satisfy the requirements.

[1] These points were discussed and decided in meetings and Slack. Summarizing here for future reference. 👍

Reply:
`z-1` was introduced in #14052 to replace `z-index: 1;` I missed this during review because I thought `z-1` was part of the defaults (good catch @pdehaan !) and a direct conversion. @xlisachan I don't know how necessary it is though or whether it even needs to be replaced with another value (the PR was merged 2 months ago and I haven't seen any issue).

Reply:
@xlisachan I've gone ahead and removed the custom config for `z-1000` and `z-9999` from TW config, and removed the one instance of `z-1` from SubscriptionTitle. Any objections?

Reply:
(apologies for the delayed response, apparently I do not get notifications when tagged - looking into this)

@vpomerleau, aligned with taking out `z-1`. We use `z-[1000]` in the `PaypalButton component` though and the appearance of the button changes when the value is removed. As @sardesam mentioned, this was one of the earlier tickets in which we were instructed to do a literal conversion. I will further investigate this in the AM since this is unexpected behavior. Thanks!

Reply:
> We use `z-[1000]` in the PaypalButton component though and the appearance of the button changes when the value is removed. As @sardesam mentioned, this was one of the earlier tickets in which we were instructed to do a literal conversion. I will further investigate this in the AM since this is unexpected behavior. Thanks!

Are you saying with this zIndex value removed in the config, `z-[1000]` does not apply the style? 🤔 Worst case scenario, we could just have this value present in payments' TW config, rather than the global one in `fxa-react`.

Reply:
Restored the custom z-index config, @LZoog recommends moving Payments-specific custom configuration out of the global config and into payments' TW config file in a future 1-pointer.